Cargo.toml

```diff
@@ -104,3 +105,4 @@ build-info-build = "0.0"
 # [patch.'https://github.com/WalletConnect/WalletConnectRust.git']
 # relay_rpc = { path = "../WalletConnectRust/relay_rpc" }
 # relay_client = { path = "../WalletConnectRust/relay_client" }
+# blockchain_api = { path = "../WalletConnectRust/blockchain_api" }
```

Comment:
I'm curious, do we need to maintain these lines commented out?

Reply:
So it's easy to un-comment and be able to test local changes easily